archive.php

```diff
@@ -18,7 +18,7 @@
 get_header(); ?>
 
 <div class="main-wrap" role="main">
```

Comment:
`role="main"` can be dropped from here. It belongs on the main element but we don't need it there since it's applied by default.